shard-operator/src/main/java/com/redhat/service/smartevents/shard/operator/BridgeExecutorServiceImpl.java

```diff
@@ -53,7 +53,7 @@ public class BridgeExecutorServiceImpl implements BridgeExecutorService {
     KubernetesClient kubernetesClient;
 
     @Inject
-    CustomerNamespaceProvider customerNamespaceProvider;
+    CustomerBridgeNamespaceProvider customerBridgeNamespaceProvider;
```

Comment:
tl;dr; I added `BridgeId` to the namespace provider (in addition to `CustomerId`).

Did we want to retain the use of `CustomerId` (or just use `BridgeId`)?